Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -117,25 +118,24 @@ function UnbindCurrentSource()
 #
 # MAIN SCRIPT
 #
-GetDistro
 case $DISTRO in
-	redhat_6 | centos_6 | redhat_7 | centos_7)
+	redhat_6 | centos_6 | redhat_7 | centos_7 | debian*)
```

Comment:
How come debian suddenly doesn't support unbind?

Reply:
This is the same question I have. I looked into the change. The available_clocksource has 3 values "hyperv_clocksource_tsc_page hyperv_clocksource acpi_pm", but the test code looks for acpi_pm always but Debian presented hyperv_clocksource at the end. I don't think Debian doing the same pattern as other distros. The database shows we have the result with the customized kernel most of the time.  Even we ran it in Debian 9 latest image but used the custom kernel.  I propose here we set the test principle working in the latest Azure image, not the customized image or kernel. 
Does it sound good?